<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>welcome_screen</class>
 <widget class="QMainWindow" name="welcome_screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(183, 229, 176);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="header_lb">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Welcome to Password Encryption Tool(PET)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generate_keys_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>140</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Keys</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="encryption_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Encryption</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="decryption_btn">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>280</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Decryption</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="htu_lb">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>How to use:</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>251</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string>Step 1: Generate Keys and store them safely on
 your local machine
Step 2: Encrypt/Decrypt your Message
Step 3: Save the output for later use!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
